tests/unit_tests/core/fake_device_module.py

```diff
@@ -14,6 +18,45 @@ def fake_motor_x() -> EpicsMotor:
     return _mock_with_name("motor_x")
 
 
+class Device_a(StandardReadable):
```

Comment:
Could: `PascalCase`, see https://peps.python.org/pep-0008/#class-names

```suggestion
class DeviceA(StandardReadable):
```

Reply:
good catch